controller/konnect/server/region.go

```diff
@@ -0,0 +1,50 @@
+package server
+
+import (
+	"fmt"
+	"regexp"
+)
+
+// Region represents a Konnect region.
+type Region string
+
+func (r Region) String() string {
+	return string(r)
+}
+
+// These are the well-known Konnect regions. The list is not exhaustive.
+const (
+	// RegionGlobal represents the global region.
+	RegionGlobal Region = "global"
+	// RegionEU represents the European region.
+	RegionEU Region = "eu"
+	// RegionUS represents the United States region.
+	RegionUS Region = "us"
+	// RegionAU represents the Australian region.
+	RegionAU Region = "au"
+	// RegionME represents the Middle East region.
+	RegionME Region = "me"
+	// RegionIN represents the Indian region.
+	RegionIN Region = "in"
```

Comment:
These are now unused (apart from some usage in tests) so I guess we can get rid of them. WDYT?